FEXCore/Source/Interface/Core/OpcodeDispatcher/AVX_128.cpp

```diff
@@ -948,4 +948,73 @@ void OpDispatchBuilder::AVX128_VectorScalarInsertALU(OpcodeArgs) {
   AVX128_StoreResult_WithOpSize(Op, Op->Dest, RefPair {.Low = Result_Low, .High = High});
 }
 
+Ref OpDispatchBuilder::AVX128_VFCMPImpl(size_t ElementSize, Ref Src1, Ref Src2, uint8_t CompType) {
+  Ref Result {};
+  switch (CompType) {
```

Comment:
Unless there's a strong reason otherwise, I'd rather switch on `CompType & 7` which lets you delete 2/3 the code here.

Reply:
Fixed